### ISSUE TITLE
Please Disable Trestle In Test Env

### ISSUE THREAD

**Issue by bendangelo** [TrestleAdmin/trestle#482]
I noticed if my model has a nomethod error on load. Trestle will catch this error and output it's own error saying it can't load. This is fine in dev but when running specs this ends up hiding errors.

Can trestle lets these errors through?

Example:
```
NameError:
  Unable to find model ExampleSentence. Specify a different model using Trestle.resource(:example_sentences, model: MyModel)
# ./app/admin/example_sentences_admin.rb:1:in `<main>'
# ./config/environment.rb:5:in `<top (required)>'

```

**Comment by spohlenz** [comment]
Thank you for reporting @bendangelo. I definitely agree that Trestle has previously been too sensitive to errors on initial load, even in development mode, but especially in test mode as you've highlighted.

Could you please let me know how you go with the fix in #483 and I'll likely push a new gem release this week if all is well.

**Comment by bendangelo** [comment]
Hi, I just tested and it looks good to me. Thanks!
